poetry.lock

```diff
@@ -332,7 +332,7 @@ version = "1.17.4"
 description = "NumPy is the fundamental package for array computing with Python."
 category = "main"
 optional = false
-python-versions = ">=3.6"
+python-versions = ">=3.5"
```

Comment:
Is this change on purpose?

Reply:
As discussed in Slack: Yes this was done automatically by poetry after running `poetry lock --no-update` to fix the hashes. Previously the lock file contained the hashes for the wrong numpy version.